size_benchmarks/earlyflush/sizes.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+import argparse
+import concurrent.futures
+import glob
+import itertools
+import io
+import os
+import subprocess
+
+
+# binjs_generate_prediction_tables takes a dictionary output path
+# whereas binjs_encode takes an input dictionary file; we just work
+# with paths and hard-code the filename the same way as
+# binjs_generate_prediction_tables.
+def dict_path_to_filename(path):
+  return os.path.join(path, 'dict.entropy')
+
+
+def generate_dictionary(args):
+  input_files = args.input_files
+  output_dictionary = args.dictionary
+  cmd = ['../../target/release/binjs_generate_prediction_tables',
+         '-o', output_dictionary, '-i'] + [f.name for f in input_files]
+  subprocess.run(cmd)
+  with open(dict_path_to_filename(output_dictionary), 'rb') as d:
+    dict_size = brotli_compressed_size(d)
+  print('brotli compressed dictionary size:', dict_size)
+
+
+def binjs_compressed_size(input_file, dictionary):
+  if not os.path.exists(dictionary):
+    raise Exception('dictionary file {} does not exist'.format(dictionary))
+  with subprocess.Popen(
+      ['../../target/release/binjs_encode', 'advanced', 'entropy',
+       '--dictionary', dictionary],
+      stdin=subprocess.PIPE,
+      stdout=subprocess.PIPE) as proc:
+    stdout, stderr = proc.communicate(input_file.read())
+    if stderr:
+      raise Exception(stderr)
+    return len(stdout)
+
+
+def brotli_compressed_size(inp):
```

Comment:
e.g. clarify that it's the compressed size of the js file, not the binjs file, right?